sdk/resourcemanager/Azure.ResourceManager.Core/src/Response/VoidArmResponse.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    internal class VoidArmResponse : ArmResponse
```

Comment:
same as above ArmVoidResponse?  this is just a nit, though, and opinions may vary.